common/include/sim/common.hpp

```diff
@@ -15,6 +29,67 @@ using VirtAddr = RegValue;
 
 using InstrCode = uint32_t;
 
+namespace bit {
+
+using BitIdx = size_t;
+using BitSize = size_t;
+
+static constexpr size_t BYTE_SIZE = 8;
+static_assert(CHAR_BIT == BYTE_SIZE);
+
+// Return ones mask for given unsigned integer type
+template <class UInt> constexpr UInt ones() noexcept {
```

Comment:
please change func name
For exmaple, onesMask

Reply:
Done